### ISSUE TITLE
Issue with execute() expecting an integer return in symfony 5

### ISSUE THREAD

**Issue by matijaboban** [Comcast/php-legal-licenses#14]
HI @orediggerco,
it seems that the update to symfony/console and symfony/process update to ~5.0 in v1.1.3 (#12 ) introduced an issue related to expected execute() output, as it is expected as integer now.

This is causing the following issue when running the command:
```
Return value of "Comcast\PhpLegalLicenses\Console\GenerateCommand::execute()" must be of the type int, NULL returned. 
```
I've taken a quick look at the documentation at Symfony console [docs](https://symfony.com/doc/current/console.html) and opened a simple PR (#13 ) to solve this.

Please note that as the project doesn't seem to have exposed unit tests, I haven't included any in the PR.

**Comment by orediggerco** [comment]
Thanks for catching this @matijaboban! I'll review the PR this morning.

**Comment by orediggerco** [comment]
Closed with #13
